src/modules/components/banners/notification/notification.component.ts

```diff
@@ -3,11 +3,12 @@ import { Component } from '@govuk/angularjs-devtools';
 @Component({
   bindings: {
     type: '@',
-    showIcon: '<'
+    showIcon: '<',
+    flow: '<'
   },
   transclude: true,
   template: `
-    <div class="pdk-notification pdk-notification-{{$ctrl.type}}" role="alert">
+    <div class="pdk-notification pdk-notification-{{$ctrl.type}} {{$ctrl.flow ? 'flow' : ''}}" role="alert">
```

Comment:
for this case I'd use ng-class, otherwise you are adding an empty space for no reason in the class